.github/workflows/container-image.yml

```diff
@@ -77,6 +79,8 @@ jobs:
           context: .
           file: ./Dockerfile
           platforms: linux/amd64
+          sbom: true
```

Comment:
I think this is not enough for creating the SBOMs for multistage images. 
For multistage images, we need to add `BUILDKIT_SBOM_SCAN_STAGE=true` globally, for all the stages we want to scan, in Dockerfile.
See https://docs.docker.com/build/metadata/attestations/sbom/#arguments.